### PR TITLE
login limiter is being breached during the test trial

### PR DESCRIPTION
**Pull request by eoliveros**
- [x] changed the login and register limiter from 10/hr to 20/min

### CODE REVIEW

**Review by djpnewton — CHANGES_REQUESTED** [review]
the html login limits dont need to be changed (we dont expect many people to be logging in via the backend forms).

instead you need to change the limits in api_endpoint.py (including the one requested here https://github.com/zap-me/ops/issues/180#issuecomment-1203442910)

File: src/security.py [hunk 1]
```diff
@@ -154,7 +154,7 @@ def _limit_view_function(view_name, limit_func):
     app.view_functions[view_name] = limit_func(app.view_functions[view_name])
 _limit_view_function('security.login', limiter.limit('10/hour'))
 if app.config['SECURITY_REGISTERABLE']:
-    _limit_view_function('security.register', limiter.limit('10/hour'))
```
Comment (djpnewton) [inline src/security.py:157]: this is not required as *only* admins will use the web portal

**Review by djpnewton — CHANGES_REQUESTED** [review]

File: src/security.py (hunk 1 above)
Comment (eoliveros) [reply src/security.py:157]: so should i delete the "limiter.limit('10/hour')" part of the security.login?

Comment (djpnewton) [reply src/security.py:157]: you should not change the limits on security.register/login

Comment (eoliveros) [reply src/security.py]: changed it back security.register to 10/hr

**Review by djpnewton — CHANGES_REQUESTED** [review]
you need to change the commit message to not have stuff that is not represented in the final changes.

eg `and reverted changes on security endpoint`. If you look at the commit changes there are no changes to the secuirty enpoint so talking about changes to the security endpoint can only confuse a later reader

**Pull request merged**